### PR TITLE
Code cleanup. Cleaned up unnecessary null checks and unused parameters.

### PR DESCRIPTION
**Pull request by glang**
Code cleanup. Cleaned up unnecessary null checks and unused parameters.

### CODE REVIEW

File: src/main/java/com/sforce/cd/apexUnit/client/utils/ApexClassFetcherUtils.java [hunk 1]
```diff
@@ -359,7 +359,7 @@ public static String fetchAndAddToMapApexClassIdBasedOnName(PartnerConnection co
 			ApexUnitUtils.shutDownWithDebugLog(e, ConnectionHandler
 					.logConnectionException(e, connection, soql));
 		}
-		if (queryResult == null || (queryResult.getRecords() != null && queryResult.getRecords().length <= 0)) {
+		if (queryResult == null || queryResult.getRecords().length <= 0) {
```
Comment (glang) [inline src/main/java/com/sforce/cd/apexUnit/client/utils/ApexClassFetcherUtils.java]: This is a mistake. I will fix this tomorrow.

**Comment by adarsh-ramakrishna-sfdc** [comment]
@glang generateHTMLReportTest failed on Travis. Did you test this locally? and on the internal jenkins instance?

**Comment by adarsh-ramakrishna-sfdc** [comment]
As long as you confirm that all tests pass, :+1 from me

**Comment by glang** [comment]
@adarsh-ramakrishna-sfdc the generateHTMLReportTest is a strange one. It fails and passes randomly on Travis as far as I can tell. I have tested it locally a couple of times and it always passes. I will check on jenkins on Monday, as I think my stage-jenkins job that I normally use has been renamed or removed. I'll ask @pmedapuram about that.

**Comment by adarsh-ramakrishna-sfdc** [comment]
Changes looks good to me 👍 
The second reviewer can review and merge the PR

**Comment by glang** [comment]
@pmedapuram Hi, would you mind doing the second review for this pull request for me?

**Comment by pmedapuram** [comment]
👍

**Pull request merged**